tests/pipelines/test_pipelines_depth_estimation.py

```diff
@@ -17,15 +17,7 @@
 
 from transformers import MODEL_FOR_DEPTH_ESTIMATION_MAPPING, is_torch_available, is_vision_available
 from transformers.pipelines import DepthEstimationPipeline, pipeline
-from transformers.testing_utils import (
-    is_pipeline_test,
-    nested_simplify,
-    require_tf,
-    require_timm,
-    require_torch,
-    require_vision,
-    slow,
-)
+from transformers.testing_utils import nested_simplify, require_tf, require_timm, require_torch, require_vision, slow
```

Comment:
Since you create your PR and today we removed `is_pipeline_test` as we're now passing on these tests based on directory not by ENV variable.